tests/tr1/tests/atomic/test.cpp

```diff
@@ -222,6 +222,72 @@ static void test_atomic_flag() { // test properties of atomic_flag
     CHECK_INT(value, false);
 }
 
+#if _HAS_CXX20
+// TEST C++20 SEMANTICS OF atomic_flag
+static void test_atomic_flag_cxx20() { // test properties of atomic_flag
+    STD atomic_flag flag = ATOMIC_FLAG_INIT;
+    bool value           = flag.test();
+    CHECK_INT(value, false);
+    value = flag.test();
+    CHECK_INT(value, false);
+    value = flag.test_and_set(STD memory_order_acq_rel);
+    CHECK_INT(value, false);
+    value = flag.test(STD memory_order_acquire);
+    CHECK_INT(value, true);
+    flag.clear();
+    value = flag.test();
+    CHECK_INT(value, false);
+    value = flag.test_and_set();
+    CHECK_INT(value, false);
+    value = flag.test();
+    CHECK_INT(value, true);
+    flag.clear(STD memory_order_release);
+    value = flag.test_and_set();
+    CHECK_INT(value, false);
+    value = flag.test(STD memory_order_acquire);
+    CHECK_INT(value, true);
+
+    volatile STD atomic_flag vflag = ATOMIC_FLAG_INIT;
+    value                          = vflag.test(STD memory_order_relaxed);
+    CHECK_INT(value, false);
+    value = vflag.test_and_set(STD memory_order_release);
+    CHECK_INT(value, false);
+    value = vflag.test_and_set(STD memory_order_release);
+    CHECK_INT(value, true);
+    value = vflag.test(STD memory_order_acquire);
+    CHECK_INT(value, true);
+    vflag.clear();
+    value = vflag.test_and_set();
+    CHECK_INT(value, false);
+    flag.clear(STD memory_order_relaxed);

```

Comment:
It looks like the search-and-replace of `flag` with `vflag` stopped a few lines early here.